src/functions.php

```diff
@@ -146,7 +146,7 @@ function is_last_pipeline_operator_out(array $pipeline)
 
     $lastOp = (array) $lastOp;
 
-    return key($lastOp) === '$out';
+    return in_array(key($lastOp), ['$out', '$merge']);
```

Comment:
It may not make a practical difference, but preserving `===` warrants setting `$strict` to `true`. I bring this up in case it's relevant for some future sniff (e.g. prohibiting loose equality checks across the board).

Reply:
Good point. I've added a `true` as third argument to enable strict checks.